packages/myst-transforms/src/enumerate.ts

```diff
@@ -308,6 +310,8 @@ export class ReferenceState implements IReferenceState {
       this.warnNodeTargetNotFound(node);
       return;
     }
+    // Put the kind on the node so we can use that later
+    node.kind = target.kind;
```

Comment:
@fwkoch I needed this information on the node for the docx export. i.e. what is the kind of the crossref node when you are exporting, changes the link target in word.